src/icons/fileIcons.ts

```diff
@@ -2193,5 +2193,30 @@ export const fileIcons: FileIcons = {
         '.cracorc',
       ],
     },
+    {
+      name: 'redhat',
+      fileNames: ['.s2iignore'],
+    },
+    {
+      name: 'openshift',
+      fileExtensions: ['.openshift/conf'],
+    },
+    {
+      name: 'ansible',
+      fileNames: [
+        'ansible.cfg',
+        'inventory',
+        'ansible-navigator.yml',
+        'ansible-navigator.yaml',
+        'ansible-navigator.json',
+        'execution_environment.yml',
+        'execution_environment.yaml',
+        'playbook.yml',
+        'playbook.yaml',
+        'main.yml',
```

Comment:
Thanks for the suggestions, but I always want to make sure that the file names are not too generic. Hopefully you will understand that not everyone of all VS Code users who creates a file called 'main.yml' is using it in an ansible environment. I'd recommend to remove these generic names from this MR. Sample applies for 'inventory'

Reply:
Makes sense. Main.yml could also be a github-actions file.